updater_test.go

```diff
@@ -289,7 +287,7 @@ func TestUpdaterCancel(t *testing.T) {
 }
 
 func TestUpdaterSnooze(t *testing.T) {
-	testServer := testServerForError(t, fmt.Errorf("snooze"))
```

Comment:
If we want to test the prompt action flow, we now need to go through with getting a download first.